certified-connectors/Criteo/api.properties

```diff
@@ -0,0 +1,24 @@
+{
+  "connectionParameters": {
+    "token": {
+      "type": "oauthSetting",
+      "oAuthSettings": {
+        "identityProvider": "oauth2",
+        "clientId": "criteo_client_id",
```

Comment:
not sure if I have to also put "clientSecret": ""

Reply:
@gabrigiunchi No need to add the clientSecret :-)